### PR TITLE
Feature/get all apis in a category

### PR DESCRIPTION
**Pull request by alahirajeffrey**
endpoint to get all apis in a category

### CODE REVIEW

File: src/categories/categories.service.ts [hunk 1]
```diff
@@ -23,4 +31,18 @@ export class CategoriesService {
   remove(id: number) {
     return `This action removes a #${id} category`;
   }
-}
+
+  async getAllApis(categoryId: string){
+    //check if category exists
```
Comment (TijanAyo) [inline src/categories/categories.service.ts:36]: Nice one...

You can use a try-catch block here

Comment (alahirajeffrey) [reply src/categories/categories.service.ts]: Thanks. Done

**Review by TijanAyo — DISMISSED** [review]
🚀🚀

File: src/categories/categories.controller.ts [hunk 2]
```diff
@@ -17,11 +20,17 @@ export class CategoriesController {
     return this.categoriesService.findAll();
   }
 
+  @Get('getAllApis:id')
```
Comment (johnayinde) [inline src/categories/categories.controller.ts:23]: i dont know if there are naming convention for endpoints @PhilemonBrain @Chimexx 

the endpoint name dosnt look nice

File: src/categories/categories.controller.ts [hunk 3]
```diff
@@ -17,11 +20,17 @@ export class CategoriesController {
...
+  @ApiOperation({summary:"get all apis in a particular category"})
+  findAllApis(@Param('categoryId') categoryId: string) {
```
Comment (johnayinde) [inline src/categories/categories.controller.ts:25]: u used :id as the route parameter and now using categoryId again. why?

Comment (alahirajeffrey) [reply src/categories/categories.controller.ts:25]: Mistake. It has been fixed

File: src/categories/categories.controller.ts (hunk 2 above)
Comment (alahirajeffrey) [reply src/categories/categories.controller.ts]: I have fixed the endpoint name

**Review by johnayinde — APPROVED** [review]

**Review by TijanAyo — APPROVED** [review]

**Review by johnayinde — APPROVED** [review]

**Pull request merged**